### PR TITLE
Moved aws-backup role to common - Closes #9

### LINKED ISSUE #9: Add role for aws-backup

### PR DESCRIPTION
**Pull request by psychopenguin**
Closes #9

### CODE REVIEW

File: roles/aws-backup/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+- name: Ensure /srv/backups directory exist
+  file:
+    state: directory
+    path: /srv/backups
+    owner: root
```
Comment (fchavant) [inline roles/aws-backup/tasks/main.yml]: Set permissions to `0750`.

File: roles/aws-backup/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    state: present
+    name:
+      - gnupg2
+      - awscli
+
+- name: Import operators PGP keys
+  shell: 'echo "{{ item.pgpkey }}" | gpg2 --import'
+  register: result
+  changed_when: "'unchanged: 1' not in result.stderr"
+  loop: "{{ operators }}"
```
Comment (fchavant) [inline roles/aws-backup/tasks/main.yml]: `operators` should be defined (empty) in `defaults/main.yml` with an example.

File: roles/aws-backup/defaults/main.yaml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (fchavant) [inline roles/aws-backup/defaults/main.yaml]: Missing empty line, space before comment.

File: roles/aws-backup/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+- name: Ensure /srv/backups directory exist
```
Comment (fchavant) [inline roles/aws-backup/tasks/main.yml]: Missing `---`

File: roles/aws-backup/defaults/main.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+aws_backup__backups_max_age: 14  # days
+# The operators variable is an array of users to be created on target system
+# example:
+# operators:
+# - name: username
+# hashed_password: "<SHA256 hashed password>"
```
Comment (Nazgolze) [inline roles/aws-backup/defaults/main.yaml]: It's not necessarily SHA256, just has to be in the crypt format (man 3 crypt)

File: roles/aws-backup/defaults/main.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+aws_backup__backups_max_age: 14  # days
+# The operators variable is an array of users to be created on target system
+# example:
+# operators:
+# - name: username
+# hashed_password: "<hashed password in crypt format>"
```
Comment (fchavant) [inline roles/aws-backup/defaults/main.yaml]: Shouldn't this be indented?

**Pull request merged**